doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/time.mdx

```diff
@@ -229,10 +242,10 @@ d"2021-01-01T00:00:00Z"
 
 ## `time::hour`
 
-The `time::hour` function extracts the hour as a number from a datetime.
+The `time::hour` function extracts the hour as a number from a datetime, or from the current date if no datetime argument is present.
 
 ```surql title="API DEFINITION"
-time::hour(datetime) -> number
+time::hour(option<datetime>) -> number
 ```
 The following example shows this function, and its output, when used in a [`RETURN`](/docs/surrealdb/surrealql/statements/return) statement:
```

Comment:
Ahh that's a problem with every single link at here, can I open a pull request for this? :)

Reply:
`doc-surrealdb_versioned_docs/version-latest/surrealql/statements/return.mdx` this shall be the right path for this, I believe.

Reply:
Hi @itszeeshan, you can try the paths out inside the Deploy Preview link, currently this one here: https://deploy-preview-813--surrealdb-docs.netlify.app/docs/surrealdb/surrealql/functions/database/time#timehour

Reply:
yup, it works, and totally my bad. I was looking at the repo, and noticed that no docs folder was present - so thought to point out. :)